### PR TITLE
[ready] Beepeople, pollen, family heirlom, beecommon  language .

### PR DESCRIPTION
**Pull request by DaddyIssues98**
## About The Pull Request

New race: bee people:  they were made by ARC, a bee-human hibric. they have made they own 2 lenguages, knows as "common bee" and "royal bee" (only common is in the game)

image

## Why It's Good For The Game

i like bees and nobody can stop me from doing this. 

more species and a new food type give more RP opportunitys. 

bees

## Changelog
:cl:
add: added bee specie.
add: added bee common language
add: Added pollen as food type
add: added new cubic bee family heirloom.
del: one single space in the code
tweak: moths can eat pollen
soundadd: added an attack and miss attack for bees
soundadd: added screams for bees.
imageadd: added unholy ass
spellcheck:  all my code have typos
config: makes beepeople round start race
admin: did give me headadmin permissions, as im a admin now
/:cl:

### CODE REVIEW

**Comment by DaddyIssues98** [comment]
youll need to ignore the big commented code

**Comment by BootlegBow** [comment]
based and bungapilled

File: code/modules/mob/living/carbon/human/species_types/beeperson.dm [hunk 1]
```diff
@@ -0,0 +1,82 @@
+/datum/species/human/beepeople
```
Comment (BootlegBow) [inline code/modules/mob/living/carbon/human/species_types/beeperson.dm:1]: reskinned human confirmed????

Comment (DaddyIssues98) [reply code/modules/mob/living/carbon/human/species_types/beeperson.dm:1]: they are kinda like catpeople. they dna is just a little of human, a little of bee and a ton of unestable mutadone, grind for 5 minutes and you have it

**Comment by BootlegBow** [comment]
felinid and beelinid

**Comment by Kitsunemitsu** [comment]
I recommend you taking off the hair option, but I will be testmerging.
It's all up to you.

**Comment by DaddyIssues98** [comment]
i just need some feedback from the sprites and tests and it will be full, later will add unique stuff to the bess, not now cause time

File: strings/names/bee_first.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Halictidae
+Melittidae
+Stenotritidae
+Leafcutter
+Mason
+Plasterer
+Woolcaders
+Long-horned
+Furrow
+Barry B. Benson
```
Comment (retlaw34) [inline strings/names/bee_first.txt:15]: ‎

**Review by BootlegBow — APPROVED** [review]
now we hold sakurai at gunpoint and add barry b. benson to smash

**Pull request merged**